pkg/providers/pricing/suite_test.go

```diff
@@ -252,3 +260,30 @@ var _ = Describe("Pricing", func() {
 			To(ContainElements("Linux/UNIX", "Linux/UNIX (Amazon VPC)"))
 	})
 })
+
+func getPricingEstimateMetricValue(instanceType string, capacityType string, zone string) float64 {
+	gathering, err := crmetrics.Registry.Gather()
+	Expect(err).To(BeNil())
+	var value *float64
+	for _, mf := range gathering {
+		if mf.GetName() != "karpenter_pricing_instance_estimate" {
+			continue
+		}
+		for _, m := range mf.GetMetric() {
+			_, itFound := lo.Find(m.Label, func(label client_model.LabelPair, _ int) bool {
+				return aws.StringValue(label.Name) == "node_kubernetes_io_instance_type"
+			})
+			_, ctFound := lo.Find(m.Label, func(label client_model.LabelPair, _ int) bool {
+				return aws.StringValue(label.Name) == "node_kubernetes_io_instance_type"
```

Comment:
I believe you might've copy pasta'd the wrong values:
```suggestion
				return aws.StringValue(label.Name) == "    karpenter_sh_capacity_type"
```